app/src/androidTest/java/com/example/habitsmasher/MainActivityTest.java

```diff
@@ -1982,6 +1981,83 @@ public void navigateToHabitEventViewFragment() {
         deleteTestHabit(testHabit);
     }
 
+    @Test
+    public void navigateToFollower(){
+        logInTestUser();
+
+        // go to the profile
+        _solo.clickOnView(_solo.getView(R.id.navigation_notifications));
+
+        // click on the followers button
+        _solo.clickOnView(_solo.getView(R.id.number_followers));
+
+        // click on the test user
+        _solo.clickOnText("publicTester");
```

Comment:
Extract strings out global variables.

Reply:
Didn't extract out the Public and private strings though